client/my-sites/checkout/checkout-thank-you/custom-domain-purchase-detail.jsx

```diff
@@ -55,23 +55,7 @@ const CustomDomainPurchaseDetail = ( {
 			/>
 		);
 	} else if ( ! hasDomainCredit && hasCustomDomain( selectedSite ) ) {
```

Comment:
Can we remove this `else if` condition entirely? There's a `return null;` at the end of the function which should take care of the return value for this function.

Reply:
Good call, updating that now!